### PR TITLE
[Fix]: Table page change issue

### PR DESCRIPTION
**Pull request by nxanil**
@natabal 

Fix issue #368. Please have a look.

### CODE REVIEW

File: Graphs/Table/index.js [hunk 1]
```diff
@@ -710,6 +713,8 @@ class Table extends AbstractGraph {
         }
 
         this.scroll ? this.updateTableStatus({ currentPage: this.currentPage, event: events.PAGING }) : this.updateData();
+        this.headerData = this.getHeaderData(this.getInitialSort());
+        this.updateData();
```
Comment (natabal) [inline Graphs/Table/index.js:717]: potentially you could run updateData twice: see above. In any case tested this and the solution does not work

**Comment by natabal** [comment]
@nxanil @nxjiten This solution is incomplete see my comments. Please address

**Comment by nxanil** [comment]
@natabal 

It's working fine for us as you can see below. If you are still facing same issue then we can discuss it over the call.

vssTable

**Comment by natabal** [comment]
@nxanil If the issue is fixed in the new graph please close this PR

**Pull request closed without merging**